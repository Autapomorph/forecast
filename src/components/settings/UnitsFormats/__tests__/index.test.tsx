import React from 'react';
import { shallow } from 'enzyme';

import { UnitsFormats } from '..';

describe('UnitsFormats tests', () => {
  const unitsFormats = ['format1', 'format2'];
  const props = {
    availableUnitsFormats: unitsFormats,
    selectedUnitsFormat: unitsFormats[0],
    _changeUnitsFormat: jest.fn(),
  };

  it('renders properly', () => {
    const wrapper = shallow(<UnitsFormats {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
