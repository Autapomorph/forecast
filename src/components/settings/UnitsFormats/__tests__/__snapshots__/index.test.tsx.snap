// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnitsFormats tests renders properly 1`] = `
<styles__SettingContainer>
  <styles__SettingTitle>
    Units
  </styles__SettingTitle>
  <styles__SettingContent>
    <Select
      onChange={[Function]}
      options={
        Array [
          Object {
            "label": "Metric",
            "value": "metric",
          },
          Object {
            "label": "Imperial",
            "value": "imperial",
          },
        ]
      }
      value={
        Object {
          "label": "settings.unitsFormats.type.format1",
          "value": "format1",
        }
      }
    />
  </styles__SettingContent>
</styles__SettingContainer>
`;
